eventstore/signals.py

```diff
@@ -0,0 +1,13 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from eventstore.models import OpenHIMQueue, PrebirthRegistration
+
+
+@receiver(post_save, sender=PrebirthRegistration)
+def create_user_profile(sender, instance, created, **kwargs):
```

Comment:
Should this function name be something like `queue_prebirth_registration`?

Reply:
Yes